api/files/params/upload-file-parameters.ts

```diff
@@ -44,4 +44,10 @@ export class UploadFileParameters extends BaseParameters {
         this.set("fileUri", fileUri);
         return this;
     }
+
+    setCallbackUrl(callbackUrl: string): UploadFileParameters {
```

Comment:
I like validation in job callback. https://github.com/Smartling/api-sdk-nodejs/blob/a442578c22fc1c1eeb966d71ec037b9c36ea36dc/api/jobs/params/create-update-job-base-parameters.ts#L45-L47

Reply:
Done